packages/logger/src/logger-context.ts

```diff
@@ -4,6 +4,8 @@ import { SALEOR_API_URL_HEADER, SALEOR_EVENT_HEADER } from "@saleor/app-sdk/cons
 
 export class LoggerContext {
   private als = new AsyncLocalStorage<Record<string, unknown>>();
+  // eslint-disable-next-line turbo/no-undeclared-env-vars
```

Comment:
Don't we need to add it to turbo.json for other apps like we have in avatax?

Reply:
Not sure, but it's been like that for a good while 😛 
https://github.com/saleor/apps/blob/main/packages/logger/src/logger-otel-transport.ts#L58

Reply:
I suggest we try to add it into turbo.json where needed and see 😄 - it will help us with turbo cache etc.

Reply:
We need to test it, I think there is no workspace turbo json file for packages, not sure if linter can handle this anyway. So we should probably have this env in root turbo config, and probably keep the comment

Reply:
Yeah, linter doesn't handle it at all (very strange) - the env is in avatax and products-feed. We need to keep it as it is

Reply:
Its not strage, turbo.json is in apps only, not in packages

Reply:
@Cloud11PL I see you added env var to turbo.json so this comment can be removed?

Reply:
Even with env var added linter still shows an error 😅